UndertaleModCli/Program.cs

```diff
@@ -270,6 +319,140 @@ private static int Info(InfoOptions options)
             return EXIT_SUCCESS;
         }
 
+        /// <summary>
+        /// Method that gets executed on the "dump" command
+        /// </summary>
+        /// <param name="options">The arguments that have been provided with the "dump" command</param>
+        /// <returns><see cref="EXIT_SUCCESS"/> and <see cref="EXIT_FAILURE"/> for being successful and failing respectively</returns>
+        private static int Dump(DumpOptions options)
+        {
+            Program program;
+            try
+            {
+                program = new Program(options.Datafile, options.Verbose, options.Output);
+            }
+            catch (FileNotFoundException e)
+            {
+                Console.Error.WriteLine(e.Message);
+                return EXIT_FAILURE;
+            }
+
+            // If user provided code to dump, dump code
+            if ((options.Code != null) && (options.Code.Length > 0) && (program.Data.Code.Count > 0))
+            {
+                // If user wanted to dump everything, do that, otherwise only dump what user provided
+                string[] codeArray;
+                if (options.Code.Contains(UMT_DUMP_ALL))
+                    codeArray = program.Data.Code.Select(c => c.Name.Content).ToArray();
+                else
+                    codeArray = options.Code;
+
+                foreach (string code in codeArray)
+                    program.DumpCodeEntry(code);
+            }
+
+            // If user wanted to dump strings, dump all of them in a text file
+            if (options.Strings)
+                program.DumpAllStrings();
+
+            // If user wanted to dump embedded textures, dump all of them
+            if (options.Textures)
+                program.DumpAllTextures();
+
+            return EXIT_SUCCESS;
+        }
+
+        /// <summary>
+        /// Method that gets executed on the "replace" command
+        /// </summary>
+        /// <param name="options">The arguments that have been provided with the "replace" command</param>
+        /// <returns><see cref="EXIT_SUCCESS"/> and <see cref="EXIT_FAILURE"/> for being successful and failing respectively</returns>
+        private static int Replace(ReplaceOptions options)
+        {
+            Program program;
+            try
+            {
+                program = new Program(options.Datafile, null, options.Output, options.Verbose);
+            }
+            catch (FileNotFoundException e)
+            {
+                Console.Error.WriteLine(e.Message);
+                return EXIT_FAILURE;
+            }
+
+            // If user provided code to replace, replace them
+            if ((options.Code != null) && (options.Code.Length > 0) && (program.Data.Code.Count > 0))
```

Comment:
Can be replaced to `if (options.Code?.Length > 0 && program.Data.Code.Count > 0)`